packages/style-engine/src/styles/background/index.ts

```diff
@@ -8,11 +8,7 @@ const backgroundImage = {
 	name: 'backgroundImage',
 	generate: ( style: Style, options: StyleOptions ) => {
 		const _backgroundImage = style?.background?.backgroundImage;
-		if (
-			typeof _backgroundImage === 'object' &&
-			_backgroundImage?.source === 'file' &&
-			_backgroundImage?.url
-		) {
+		if ( typeof _backgroundImage === 'object' && _backgroundImage?.url ) {
```

Comment:
Removing `_backgroundImage?.source === 'file'` check as we only care if background image is an object. There's no special processing for `file` or `theme` in the style engine.